.maintain/monitoring/alerting-rules/alerting-rules.yaml

```diff
@@ -108,6 +108,13 @@ groups:
     annotations:
       message: 'The node {{ $labels.instance }} has less than 3 peers for more
       than 15 minutes'
+  - alert: NoIncomingConnection
+    expr: increase(polkadot_sub_libp2p_incoming_connections_total[20m]) == 0
```

Comment:
Without a `for` would this not fire each time a new node starts as the first value would be `0` and there are no other values in the past `20m` in the timeseries?

Reply:
Tested this with my own server, where my node was offline for a few hours:

![image](https://user-images.githubusercontent.com/1412254/98936099-cd461e00-24e4-11eb-9c97-e4f6fd36c06f.png)

Instead of `20m` I put `6h`. In the period before I started the node again, the values were simply missing, and I assume missing values wouldn't trigger the alert.

Reply:
> Interestingly, however, between 1am (when my node crashed) and 7am, values were still present and decreasing linearly.

Just to make sure there is no confusion, the _increase_ rate decreased.